### DISCUSSION TITLE
Dynamic Forms?

### DISCUSSION THREAD

**Discussion by iway1** [iway1/react-ts-form#42] [Ideas]
Any interest in having this library be able to support dynamic forms? 

I think it would only make sense in applications where there's a combination of both explicitly defined forms and dynamic forms, but I think we could probably reuse a lot of the stuff here to easily make a dynamic form solution.

See https://github.com/iway1/react-ts-form/issues/30

**Comment by roborobs1023** [comment]
Definitely
